server/services/prison/PrisonService.test.ts

```diff
@@ -40,7 +40,7 @@ describe('PrisonService', () => {
     it('should pass on errors from the prison register service', async () => {
       prisonRegisterService.getActivePrisonsFromPrisonRegister.mockRejectedValue('some-error')
 
-      await prisonService.getSupportedPrisons('some-token').catch(error => {
+      await prisonService.getSupportedPrisons().catch(error => {
```

Comment:
The supported prisons endpoint is now public (same as prison register) because the information isn't sensitive (it's displayed on the public start page).